Makefile.toml

```diff
@@ -501,16 +501,24 @@ script = """
 
 set -e
 
-for tool in cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make cargo-upgrades typos-cli
+cargo install cargo-quickinstall --locked
+cargo quickinstall cargo-binstall
+
+for tool in cargo-llvm-cov cargo-nextest cargo-udeps cargo-hakari cargo-sort cargo-make
 do
   echo "install: $(tput setaf 4)$tool$(tput sgr0)"
-  cargo install $tool --locked
+  cargo binstall -y --no-symlinks $tool
   echo
 done
 
-echo "install: $(tput setaf 4)risinglightdb/sqllogictest-rs$(tput sgr0)"
-cargo install --git https://github.com/risinglightdb/sqllogictest-rs --bin sqllogictest
-echo
+# Tools that fallback to `cargo install` when using `cargo binstall`.
```

Comment:
I think eventually they'll be installable with binstall.

Reply:
And `cargo binstall` doesn't use `--locked` when fallbacking to `cargo install`

Reply:
Ι see